### PR TITLE
Strapi setup guide

### PR DESCRIPTION
**Pull request by TyDunn**

### CODE REVIEW

**Comment by AmanGuptAnalytics** [comment]
Yes @TyDunn, its working. Thanx.

**Comment by AmanGuptAnalytics** [comment]
In "Add credentials" there's a bit of incorrect numbering. Kindly see.

File: docs/website/docs/pipelines/strapi.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Strapi pipeline setup guide
+
+Strapi is a headless CMS (Content Management System), that allows developers to create powerful API-driven content management systems, without having to write a lot of custom code.
```
Comment (TyDunn) [inline docs/website/docs/pipelines/strapi.md:3]:
```suggestion
Strapi is a headless CMS (Content Management System) that allows developers to create powerful API-driven content management systems without having to write a lot of custom code.
```

**Comment by TyDunn** [comment]
@AmanGuptAnalytics Can you please review / approve?

**Review by AmanGuptAnalytics — CHANGES_REQUESTED** [review]
Everything is perfect, you may ignore my comment, but I think it would look good. Like it did in pipedrive docs

File: docs/website/docs/pipelines/strapi.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+domain = "domain" # please set me up!
+
+[destination.bigquery.credentials] # the credentials required will change based on the destination
+project_id = "set me up" # GCP project ID
+private_key = "set me up" # Unique private key (including `BEGINand END PRIVATE KEY`)
+client_email = "set me up" # Service account email
+location = "set me up" # Project location (e.g. “US”)
+```
+
+1. Replace `"api_secret_key"` with the API token you copied above
```
[the remark this answers is not in the document]
Comment (TyDunn) [reply docs/website/docs/pipelines/strapi.md:60]: Good idea! Adding this and then merging :)

**Pull request merged**